innvestigate/analyzer/base.py

```diff
@@ -1,23 +1,23 @@
-# Get Python six functionality:
```

Comment:
Of topic/pull request: I think we can drop python six in general as python 2 is not supported anymore.